src/common.jl

```diff
@@ -4,11 +4,11 @@ $(SIGNATURES)
 Get codes of GNSS system as a Matrix where each column
 represents a PRN.
 ```julia-repl
-julia> get_code(gpsl1)
+julia> get_code(GPSL1())
 ```
 """
 function get_codes(gnss::AbstractGNSS{T}) where T <: AbstractMatrix
-    @view gnss.codes[get_code_length(gnss) + 1:2 * get_code_length(gnss), :]
+    @view gnss.codes[1:get_code_length(gnss), :]
```

Comment:
Sorry, one last note here: I think we can return the complete matrix instead of the view.
`gnss.codes[1:get_code_length(gnss), :]` should be equal to `gnss.codes`. The view can be omitted.

Reply:
You're right, this is a leftover of the former code extension.